packages/liveblocks-client/src/immutable.ts

```diff
@@ -181,11 +188,11 @@ export function patchLiveList<T extends Lson>(
   }
 }
 
-export function patchLiveObjectKey<O extends LsonObject>(
+export function patchLiveObjectKey<O extends LsonObject, J extends Json>(
```

Comment:
Oh lol, this is catching a lot of issues in the Zustand package. Sweet, will have to take a look at fixing those first then, I guess.